bin/gitea-push.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+. bin/common.sh
+[ "$CI" = 'true' ] && exit 1
+set -eo pipefail
+prepare_crypt
+readonly values=$(hf_values)
+readonly gitea_enabled=$(echo "$values" | yq r - 'charts.gitea.enabled')
+[ "$gitea_enabled" != "true" ] && err "Gitea is disabled" && exit 1
+
+readonly gitea_url=$(echo "$values" | yq r - 'cluster.domain')
+readonly gitea_password=$(echo "$values" | yq r - 'charts.gitea.admin.password')
+readonly gitea_user='otomi'
+readonly gitea_repo='values'
+readonly remote_name='origin'
+
+cd $ENV_DIR
+
+# Initialize as clean slate
+if [ ! -d .git ]; then
+  git init
+fi
+# Check if gitea is origin, otherwise stop
+if [ $(git config remote.$remote_name.url) ] && [ "gitea.$gitea_url" != $(git config remote.$remote_name.url | cut -d@ -f2 | cut -d/ -f1 | cut -d: -f1) ]; then
+  err "Another origin already exists, not using gitea"
+  exit 1
```

Comment:
Since you aded this script to `bin/deploy.sh` I don't expect to see any errors when foreseen effects occur. Don't make the script exit with non-zero when we anticipate certain behaviour and don't deem it an error.